runtime/server/queries_columns.go

```diff
@@ -479,11 +488,17 @@ func (s *Server) GetTimeRangeSummary(ctx context.Context, request *runtimev1.Get
 		if err != nil {
 			return nil, err
 		}
-		summary.Min = timestamppb.New(rowMap["min"].(time.Time))
-		summary.Max = timestamppb.New(rowMap["max"].(time.Time))
-		summary.Interval, err = handleInterval(rowMap["interval"])
-		if err != nil {
-			return nil, err
+		if v := rowMap["min"]; v != nil {
+			summary.Min = timestamppb.New(v.(time.Time))
+			summary.Max = timestamppb.New(rowMap["max"].(time.Time))
+			summary.Interval, err = handleInterval(rowMap["interval"])
+			if err != nil {
+				return nil, err
+			}
+		} else {
+			summary.Min = timestamppb.Now()
+			summary.Max = timestamppb.Now()
```

Comment:
Shouldn't they be `nil` in this case?

Reply:
I wonder what should be the behaviour of functions that depend on it, let's consider we need to return an aggregation on an empty table and the time range object contains null values, should we return timeseries values array as nil or arrays should be empty?